Source/JavaScriptCore/dfg/DFGOperations.cpp

```diff
@@ -2506,6 +2506,43 @@ JSC_DEFINE_JIT_OPERATION(operationStringValueOf, JSString*, (JSGlobalObject* glo
     return nullptr;
 }
 
+
+enum class StringReplaceSubstitutions : bool { Yes, No };
+template<StringReplaceSubstitutions substitutions>
+static ALWAYS_INLINE JSString* stringReplaceStringString(JSGlobalObject* globalObject, JSString* stringCell, String&& string, String&& search, String&& replacement)
```

Comment:
nit: These shouldn't be rvalue references. I'd go with either `String` or `const String&`

Reply:
Changed.